erpnext/templates/pages/projects.py

```diff
@@ -22,6 +22,8 @@ def get_context(context):
 	project.timesheets = get_timesheets(project.name, start=0,
 		search=frappe.form_dict.get("search"))
 
+	if project_user.view_attachments == 1:
```

Comment:
Can be written as `if project_user.view_attachments:`

Reply:
Thanks for the tip